fala/apps/adviser/views.py

```diff
@@ -43,33 +42,84 @@ class AccessibilityView(TemplateView):
 class SearchView(ListView):
     def get(self, request, *args, **kwargs):
         self._form = AdviserSearchForm(data=request.GET or None)
-        self._data = self._form.search()
+        if self._form.is_valid():
+            self._region = self._form.region
+            if self._region == Region.ENGLAND_OR_WALES:
+                self._data = self._form.search()
+            else:
+                self._data = {}
+        else:
+            self._data = {}
         return super().get(self, request, *args, **kwargs)
 
     # Errors from LaaLaaAPI manifest as _data being falsey, so we bounce back to search page
```

Comment:
I think this comment is now out of date as you now use `self._form.is_valid()` and `self._region` instead, which is a really nice change